TestModels/Aggregate/src/Index.dfy

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 include "SimpleAggregateImpl.dfy"
 
-module SimpleAggregate refines AbstractSimpleAggregateService {
+module {:extern "simple.aggregate.internaldafny"} SimpleAggregate refines AbstractSimpleAggregateService {
```

Comment:
Is this a breaking change for existing projects?

Reply:
Nope! This is the current state of main-1.x.
I'm putting the extern names back in the python-reviewed branch so that when I merge this branch in, there is no diff between its Dafny source and main's.

Reply:
Ah I actually missed that this is not targeting `main-1.x` itself.